recipe/meta.yaml

```diff
@@ -1,208 +1,219 @@
-# Make sure the component versions are consistent with the SDK version!
-{% set version = "24.08.0.5" %}
```

Comment:
FYI, this variable is called "version" for a reason. The CF bot only recognizes a few variables when attempting to auto-update the version, so renaming it to something else means we'll drop the bot support (which might be fine, I dunno).

Reply:
This is a good point!